Cargo.toml

```diff
@@ -15,7 +15,7 @@ authors = [
   "Jonathan Klimt <jonathan.klimt@rwth-aachen.de>",
   "Jens Breitbart <jbreitbart@gmail.com>",
   "Reto Achermann <achreto@gmail.com>",
-  "woppopo <woppopo@protonmail.com>",
```

Comment:
Just checking this isn't an accidental deletion: this is you but different email so that's why it got removed?